pandas/tests/arrays/test_array.py

```diff
@@ -36,8 +36,30 @@
 
     # Datetime (naive)
     ([1, 2], np.dtype('datetime64[ns]'),
-     PandasArray(np.array([1, 2], dtype='datetime64[ns]'))),
-    # TODO(DatetimeArray): add here
+     pd.arrays.DatetimeArray._from_sequence(
+         np.array([1, 2], dtype='datetime64[ns]'))),
+
+    (np.array([1, 2], dtype='datetime64[ns]'), None,
+     pd.arrays.DatetimeArray._from_sequence(
+         np.array([1, 2], dtype='datetime64[ns]'))),
+
+    (pd.DatetimeIndex(['2000', '2001']), np.dtype('datetime64[ns]'),
```

Comment:
this one also with a dtype of None ?

Reply:
351c781c55b5391833ab8cccf23669226cc8de52